Commands/SitesMassWordPressUpdateCommand.php

```diff
@@ -438,37 +441,511 @@ private function update($args, $assoc_args) {
         'name' => $name,
       ));
     }
+
+    // Doing auto-deploy
+    if ( $proceed && ( $this->isAutoDeploy() || $this->yamlGetSiteSetting( $name, 'auto-deploy' ) ) ) {
+
+      $autoDeployTest = $this->auto_deploy( $name, 'test' );
+
+      $autoDeployLive = $this->auto_deploy( $name, 'live' );
+
+    }
+
+  }
+
+
+
+
+  /**
+   * Perform upstream updates on site.
+   *
+   * @param string $siteName
+   *   The name of the target site
+   * @param string $toEnv
+   *   Target site environment to deploy to
+   * @param string $apply
+   *   Apply upstream if true, inspect if false
+   * @param array $args
+   *   Additional arguments ( accept-upstream : bool, updatedb : bool )
+   */
+  private function upstream_update( $siteName, $toEnv, $apply, $args = array() ) {
+
+    $command =  'echo y | terminus site upstream-updates';
+
+    $command .= ( $apply ? ' apply' : ' list' );
+
+    $command .= ' --site=' . $siteName;
+
+    $command .= ' --env=' . $toEnv;
+
+    $default_args = array(
+      'accept-upstream' => true,
+      'updatedb'        => true,
+    );
+
+    $args = array_merge( $default_args, $args );
+
+    if ( $args ) {
+     
+      foreach ( $args as $key => $value ) {
+
+        switch ( $key ) {
+
+          case 'accept-upstream':
+            $command .= ( $value ? ' --accept-upstream' : '' );
+            break;
+
+          case 'updatedb':
+            $command .= ( $value ? ' --updatedb' : '' );
+            break;
+
+        }
+
+      }
+
+    }
+
+    $upstream = $this->execute( $command, true, true, true );
+
+    return $upstream;
+
+  }
+
+
+
+
+  /**
+   * Perform the updates on a specific site and environment.
+   *
+   * @param string $siteName
+   *   The name of the target site
+   * @param string $toEnv
+   *   Target site environment to deploy to
+   * @param array $args
+   *   Additional arguments ( cc : bool, updatedb : bool, note: string )
+   */
+  private function auto_deploy( $siteName, $toEnv, $args = array() ) {
+
+    $fromEnv = 'dev';
+
+    $command =  'terminus site deploy';
+
+    $command .= ' --site=' . $siteName;
+
+    $command .= ' --env=' . $toEnv;
+
+    $default_args = array(
+      'cc'        => true,
+      'note'      => 'Deployed files from '. $fromEnv .' to '. $toEnv .'.',
```

Comment:
@Sean12mps  Its an small change but I think important, when you deploy to `live` its actually being deployed from test, not dev. So we should test that description.